configs/components/ruby.rb

```diff
@@ -4,9 +4,9 @@
   pkg.url "https://cache.ruby-lang.org/pub/ruby/2.3/ruby-#{pkg.get_version}.tar.gz"
 
   if platform.is_windows?
-    pkg.add_source "http://buildsources.delivery.puppetlabs.net/windows/elevate/elevate.exe", sum: "bd81807a5c13da32dd2a7157f66fa55d"
+    pkg.add_source "file://resources/files/windows/elevate.exe", sum: "bd81807a5c13da32dd2a7157f66fa55d"
```

Comment:
Does it matter whether or not it's in the vanagon repo? we have been distributing it as part of the package for a while now, going back to puppet_for_the_win, and it's been part of the license file: https://github.com/puppetlabs/puppet_for_the_win/blob/be067d5c6456dba22f30938abaeeded29004bb3b/conf/windows/stage/misc/LICENSE.rtf#L12